scripts/fast-reboot

```diff
@@ -476,10 +478,16 @@ debug "Stopped  syncd ..."
 # We call `docker kill ...` to ensure the container stops as quickly as possible,
 # then immediately call `systemctl stop ...` to prevent the service from
 # restarting the container automatically.
+debug "Stopping all remaining containers ..."
 for CONTAINER_NAME in $(docker ps --format '{{.Names}}'); do
-    docker kill $CONTAINER_NAME > /dev/null
+    CONTAINER_STOP_RC=0
+    docker kill $CONTAINER_NAME &> /dev/null || CONTAINER_STOP_RC=$?
```

Comment:
You could combine L483-484 into one:
```
CONTAINER_STOP_RC=$(docker kill $CONTAINER_NAME &> /dev/null)
```

Reply:
I though this would cause failure quit when docker kill fails?

Reply:
We are running with -e. Unfortunately what you proposed would cause failure abort.

Reply:
Right. I forgot about `-e` :-(